pandas/tests/indexes/datetimes/test_tools.py

```diff
@@ -270,6 +270,38 @@ def test_to_datetime_utc_is_true(self):
         expected = pd.DatetimeIndex(data=date_range)
```

Comment:
You mentioned previously to move these tests to their own function. Is that what you were referring to?